src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt

```diff
@@ -261,17 +265,24 @@ class ComCaseloadService(
     }
   }
 
-  private fun buildVaryCaseload(managedOffenders: List<ManagedCase>): List<ManagedCase> = managedOffenders.filter { offender ->
-    offender.licences.any { licence ->
-      licence.licenceStatus in listOf(
-        ACTIVE,
-        VARIATION_IN_PROGRESS,
-        VARIATION_SUBMITTED,
-        VARIATION_APPROVED,
-        VARIATION_REJECTED,
-      ) ||
-        licence.isReviewNeeded
-    }
+  private fun getComDetails(casesToLicences: Map<ManagedOffenderCrn, LicenceSummary>): Map<String?, ProbationPractitioner> {
+    val comUsernames = casesToLicences.mapNotNull { (_, licence) -> licence.comUsername }.distinct()
+    val coms = deliusApiClient.getStaffDetailsByUsername(comUsernames)
+    return casesToLicences.map { (case, licence) ->
+      val com = coms.find { c -> licence.comUsername?.lowercase() == c.username?.lowercase() }
+      when {
+        com != null -> case.crn to ProbationPractitioner(
+          com.code,
+          name = com.name?.fullName(),
+          staffUsername = com.username,
+        )
+        case.staff == null || case.staff.unallocated == true -> case.crn to ProbationPractitioner()
```

Comment:
Presumably this is existing behaviour? an empty ProbationPractiioner record?
Just wondering if records with this get filtered out at some point?